edgelet/build/linux/package-mariner.sh

```diff
@@ -11,12 +11,12 @@ export TZ=UTC
 
 # need to use preview repo for the next 2 weeks untill mariner 2.0 gets moved to prod
 case "${MARINER_RELEASE}" in
-    '1.0-stable')
+    '1.0-'*)
```

Comment:
please change this condition to '1.0'*) the specific tag versions so not have a "-" after the first 2 version numbers